src/control/XournalMain.cpp

```diff
@@ -631,6 +635,11 @@ auto XournalMain::run(int argc, char** argv) -> int {
                            "                                 The exported file has transparent or white background,\n"
                            "                                 depending on what its format supports\n"),
                          0},
+            GOptionEntry{"export-layers", 0, 0, G_OPTION_ARG_NONE, &app_data.exportLayers,
+                         _("Export layers as pages\n"
+                           "                                 Layers are rendered successively as different pages,\n"
+                           "                                 suitable for presentation\n"),
```

Comment:
I find the description misleading.

Reply:
I see what you mean, each time I wrote this I had trouble finding the best way to express it, and this holds for variable names too